CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.3.0 (2019-07-02)
+
+- Added support for `@skyux-sdk/builder@^3`. [#63](https://github.com/blackbaud/skyux-datetime/pull/63)
```

Comment:
Is this version correct? It looks like we've added support specifically for 3.7.0.